### PR TITLE
Config Modules - blue bar with Fieldset Description

### PR DESCRIPTION
**Pull request by pe7er**
Most **Component Configuration Options** have under the option tab **a blue bar with a description**
### Summary of Changes

This PR **adds a blue bar** with description **to Modules: Options**.
#### Before the PR:

backend: System > Global Configuration > (in Left menu) Modules > [**General**]

info-label-options-modules-general-before
#### After the PR:

backend: System > Global Configuration > (in Left menu) Modules > [**General**]

info-label-options-modules-general-after

### CODE REVIEW

File: administrator/components/com_modules/config.xml [hunk 1]
```diff
@@ -2,7 +2,9 @@
 <config>
     <fieldset
         name="modules"
-        label="COM_MODULES_GENERAL">
+        label="COM_MODULES_GENERAL"
+		description="COM_MODULES_GENERAL_FIELDSET_DESC">
```
Comment (zero-24) [inline administrator/components/com_modules/config.xml]: can we remove the extra tab?

Comment (jeckodevelopment) [inline administrator/components/com_modules/config.xml]: done :D

**Comment by zero-24** [comment]
Please double check @jeckodevelopment

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on f1dda9ac232c8be3073424337fa30cb19b032f4d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12090">issues.joomla.org/joomla-cms/12090</a>.</sub>

**Comment by zero-24** [comment]
sorry @pe7er i have just fixed the file that was completly wrong formeatted so you had no chance to do it right now it is correct.

**Review by zero-24 — APPROVED** [review]

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 6429159b483fd85a60b032bc457ff6f24f4eb444<br /><br />I have also checked code changes.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12090">issues.joomla.org/joomla-cms/12090</a>.</sub>

**Comment by zero-24** [comment]
@brianteeman please have a look here :)

**Comment by brianteeman** [comment]
OK by me

**Comment by zero-24** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12090">issues.joomla.org/joomla-cms/12090</a>.</sub>

**Comment by zero-24** [comment]
As we are in language freez we need to push that too 3.6.4

**Pull request merged**